src/bindings/python/src/pyopenvino/core/properties/properties.cpp

```diff
@@ -29,7 +29,7 @@ void regmodule_properties(py::module m) {
     wrap_property_RW(m_properties, ov::compilation_num_threads, "compilation_num_threads");
     wrap_property_RW(m_properties, ov::affinity, "affinity");
     wrap_property_RW(m_properties, ov::force_tbb_terminate, "force_tbb_terminate");
-    wrap_property_RW(m_properties, ov::denormals_optimization, "denormals_optimization");
+    wrap_property_RW(m_properties, ov::intel_cpu::denormals_optimization, "denormals_optimization");
```

Comment:
@jiwaszki what is the proper name should be used for `"denormals_optimization"`? now, it's not generic property. Do we have naming convention policy for device specific properties?

Reply:
I see device specific properties for other plugins, [ov::intel_gpu](https://github.com/openvinotoolkit/openvino/blob/master/src/inference/include/openvino/runtime/intel_gpu/properties.hpp), [ov::intel_gna](https://github.com/openvinotoolkit/openvino/blob/master/src/inference/include/openvino/runtime/intel_gna/properties.hpp), [ov::intel_myriad](https://github.com/openvinotoolkit/openvino/blob/master/src/inference/include/openvino/runtime/intel_myriad/myriad_properties.hpp) is used. Shouldn't we align the behavior across plugins? If decide update, we need update together to have similar namespace and interface to users?

Reply:
GIven we already passed CF milestone it is better not to refactor generic naming/namespaces convention tight now. I would recommend to leave "ov::intel_cpu" since it is fully aligned with other devices.

Reply:
ok, updated!